core/src/main/java/org/apache/accumulo/core/client/summary/Summary.java

```diff
@@ -78,27 +80,38 @@ public long getLarge() {
     }
 
     /**
+     * @return The number of files that were deleted after the summary retrieval operations started. This is a rare race condition where a compaction causes a
+     *         file to be deleted while retrieving summaries. When this happens, the file that replaced the deleted file can not be used because it may contain
+     *         duplication summary information for other files. Avoiding this race condition would be expensive, so reporting it was chosen. If this condition
+     *         must be avoided, then compactions must be stopped. Compactions could be stopped on a cloned table to avoid this.
+     */
+    public long getDeleted() {
+      return deleted;
```

Comment:
It was not onerous to compute.  Special action needs to be taken on file not found anyway (do not want to retry).  In addition to ignoring the file, just need to report the info back.  Reporting back was not too bad as there were already other file stats.

I could not figure out to write a test for it.  I tested it manually.